cdk/lib/stripe-patrons-data.ts

```diff
@@ -35,7 +35,7 @@ class StripePatronsDataLambda extends GuScheduledLambda {
           alarmDescription: `Triggers if there are errors from ${appName} on ${scope.stage}`,
           snsTopicName: "reader-revenue-dev",
           toleratedErrorPercentage: 1,
-          numberOfMinutesAboveThresholdBeforeAlarm: 1,
+          numberOfMinutesAboveThresholdBeforeAlarm: 46, // The lambda runs every 15 mins so alarm if it fails 3 times in a row
```

Comment:
are you sure this will work? if it's running every 15 minutes, then it will only be above the threshold for 3 minutes in any 46 minute period and the rest will be missing data points.
I don't know what this CDK translates into but I've had enough fun with cloudwatch to be concerned :)